Apps/phthreatgrid/threatgrid_connector.py

```diff
@@ -408,7 +408,8 @@ def _query_file(self, param):
             'osver': '',
             'source': '',
             'vm': param.get('vm', ''),
-            'playbook': param.get('playbook', 'default')
+            'playbook': param.get('playbook', 'default'),
+            'sample_password': param.get('file_password')
```

Comment:
Api expects the key name to be as `sample password` in the request body. Thats why we have written key name as `sample password`

Reply:
Yes we can use the same naming convention.

Reply:
In that case, should we name the param `sample_password` in the config too? This is so that the name will be consistent and easy to refer to in the API doc.

(i.e. ` 'sample_password': param.get('sample_password')` )